app-i18n/fbterm/fbterm-1.7_p20190503.ebuild

```
@@ -0,0 +1,67 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools fcaps toolchain-funcs
+
+DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
+HOMEPAGE="https://github.com/gjedeer/fbterm"
+
+MY_COMMIT="ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"
+SRC_URI="https://github.com/gjedeer/fbterm/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/fbterm-${MY_COMMIT}"
```

You can use vcs-snapshot eclass.

I'd prefer to declare `MY_COMMIT` between `inherit` and `DESCRIPTION`, and as `EGIT_COMMIT` for this.

```suggestion
inherit autotools fcaps toolchain-funcs vcs-snapshot

EGIT_COMMIT="ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"

DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
HOMEPAGE="https://github.com/gjedeer/fbterm"
SRC_URI="https://github.com/gjedeer/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

---

Done. This is not git-r3 though, why EGIT_COMMIT?

---

There is no particular reason.